mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -16,20 +16,19 @@
 from . import ugettext as _
 
 # logging
-from logging import getLogger
-log = getLogger(__name__)
+import logging
+log = logging.getLogger(__name__)
 
-class VisionControllerClient(object):
-    """Mujin Vision Controller client for binpicking tasks.
-    """
+class VisionClient(object):
```

Comment:
Can we separate the name change into another MR like https://github.com/mujin/mujinvisioncontrollerclientpy/pull/39 ?